### PR TITLE
doc/dpdk: add IPS setup docs for DPDK mode v3

### PR DESCRIPTION
**Pull request by lukashino**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [X] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [X] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [X] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/issues/5511) ticket:

Follow-up of #7869  

Describe changes:
- grammar fixes and Jul's comments

### CODE REVIEW

File: doc/userguide/setting-up-ipsinline-for-linux.rst [hunk 1]
```diff
@@ -276,3 +278,71 @@ and eBPF load balancing looks like the following: ::
...
+
+- ``copy-mode`` - see Section :ref:`afp-ips-l2-mode` for more details.
+- ``copy-iface`` - see Section :ref:`afp-ips-l2-mode` for more details.
+- ``threads`` - all interface entries must have their thread count configured and paired/connected interfaces must be configured with the same amount of threads.
+- ``mtu`` - MTU must be a common on both paired interfaces.
+
+DPDK capture module also requires having CPU affinity set in the configuration file. For the best performance,
+every Suricata worker should be pinned to a separate CPU core that is not shared with any other Suricata thread
+(e.g. management threads).
+The following snippet is from the :ref:`suricata-yaml-threading` configuration section  ::
```
Comment (jufajardini) [inline doc/userguide/setting-up-ipsinline-for-linux.rst:339]: Suggestion: 
"The following snippet shows a :ref:`suricata-yaml-threading` possible configuration set-up for DPDK ::

**Review by jufajardini — COMMENTED** [review]
three inline comments, but the decision if they'll require a new version or not is yours :)

File: doc/userguide/setting-up-ipsinline-for-linux.rst [hunk 2]
```diff
@@ -276,3 +278,71 @@ and eBPF load balancing looks like the following: ::
...
+        copy-mode: ips
+        copy-iface: 0000:3b:00.1
+        mtu: 3000
+
+The previous DPDK configuration snippet outlines several things to consider:
+
+- ``copy-mode`` - see Section :ref:`afp-ips-l2-mode` for more details.
+- ``copy-iface`` - see Section :ref:`afp-ips-l2-mode` for more details.
+- ``threads`` - all interface entries must have their thread count configured and paired/connected interfaces must be configured with the same amount of threads.
+- ``mtu`` - MTU must be a common on both paired interfaces.
```
[the remark this answers is not in the document]
Comment (lukashino) [reply doc/userguide/setting-up-ipsinline-for-linux.rst:334]: indeed, but in the end, I have rather changed it to "the same". Hopefully, it is more clear.

File: doc/userguide/setting-up-ipsinline-for-linux.rst (hunk 1 above)
Comment (lukashino) [reply doc/userguide/setting-up-ipsinline-for-linux.rst:339]: You suggestion translates to:
`The following snippet shows a [Threading](http://localhost:63342/suricata/doc/userguide/_build/html/configuration/suricata-yaml.html#suricata-yaml-threading) possible configuration set-up for DPDK`

which I mildly changed to:

`The following snippet shows a possible [Threading](http://localhost:63342/suricata/doc/userguide/_build/html/configuration/suricata-yaml.html#suricata-yaml-threading) configuration set-up for DPDK IPS mode.`

**Comment by lukashino** [comment]
Continues in #7901

**Pull request closed without merging**